_opcodes/op259.html

```diff
@@ -1,13 +1,52 @@
 ---
 n: 259
-name: "Protection: Spell Trap"
+name: "Trap: Effects (by Power Level, decrementing)"
```

Comment:
I would prefer if you added the parenthesesed part to the original name. "Protection" groups a bunch of related opcodes ...

Reply:
> I would prefer if you added the parenthesesed part to the original name. "Protection" groups a bunch of related opcodes ...

Done.